org.eclipse.jdt.core.tests.model/src/org/eclipse/jdt/core/tests/dom/ASTConverter10Test.java

```diff
@@ -13,20 +13,37 @@
  *******************************************************************************/
 package org.eclipse.jdt.core.tests.dom;
 
-import junit.framework.Test;
-
-import org.eclipse.jdt.core.dom.*;
```

Comment:
Any reason for not keeping the star import as it was ?

Reply:
No, it was the auto-formatting which changed that. I can revert it if you prefer?

Reply:
If the format settings are same as what I configured in the project, we can keep the change.